docs/content/deployment/deploying_scheduler.md

```diff
@@ -36,43 +36,98 @@ steps to initialize configuration and start the Airflow scheduler.
 
-   ```bash
+5. Start the Scheduler:
+   ```
    start-airflow.sh
    ```
 
+#### Deploy with Docker Compose
+
+You can follow this guide to quickly start AIFlow and Scheduler with `CeleryExecutor` within Docker.
+
+Note: the Docker Compose below may be not enough to run production-ready Docker Compose AIFlow installation using it.
+This is truly quick-start docker-compose for you to get your hands dirty with AIFlow and `CeleryExecutor`.
+
+##### Before you begin
+
+Make sure you have Docker and Docker Compose installed on your workstation,
+otherwise please follow these steps to install the tools.
+
+1. Install [Docker Community Edition (CE)](https://docs.docker.com/engine/install/) on your workstation. Depending on the OS, you may need to configure your Docker instance to use at least 4.00 GB of memory for all containers to run properly.
+2. Install [Docker Compose](https://docs.docker.com/compose/install/) on your workstation.
+
+Run the following commands to check whether the tools are installed successfully:
+```shell script
+docker --version
+docker-compose --version
+```
+
+##### docker-compose.yaml
+To deploy AIFlow with `CeleryExecutor` on Docker Compose, you should fetch [docker-compose.yaml](http://raw.githubusercontent.com/flink-extended/ai-flow/master/docker-compose.yaml).
+
+```shell script
+curl -LfO http://raw.githubusercontent.com/flink-extended/ai-flow/master/docker-compose.yaml
+```
+
+This file contains several service definitions:
+
+* AIFlow servers - AIFlow Server, Notification Server and Scheduler deployed in one container.
+* Workers - The workers who execute the tasks given by the Scheduler.
+* Redis - The broker that forwards messages from Scheduler to worker.
+* Flower -  The [flower app](https://flower.readthedocs.io/en/latest/) for monitoring the environment. It is available at http://localhost:5555.
+* MySQL - The database. Only MySQL is supported in distributed mode for now.
+* HDFS - The hadoop dfs which is used to store the source code or any resources of your projects.
+
+##### Initializing Environment
+
+This command will help start up database, redis, hdfs and 3 workers and also initialize execution environment of all servers.
+It should be executed in the directory that contains docker-compose.yaml you just downloaded.
+```shell script
+docker-compose up -d --scale airflow-worker=3
+```
+You can run ```docker ps ``` to check whether all components are started and healthy.
+
+##### Starting Services
+
+Execute following command to start up all services.
+```shell script
+docker exec ai-flow_aiflow-server_1 bash -c "start-all-aiflow-services.sh"
+```
+Once all servers started, you can visit the [Flower web](http://127.0.0.1:5555/) to view the celery workers.
+
+##### Running Example
+```shell script
+docker exec ai-flow_aiflow-server_1 bash -c "python /opt/aiflow/examples/celery/workflows/simple_workflow/simple_workflow.py"
+```
+
 (configuration)=
 
 ## Configuration
```

Comment:
This section shows a list of all the configurations we added to Airflow to make it works with AIFlow.
->
This section shows a list of configurations you may need to change to Airflow to make it works with AIFlow.